modules/ble/src/main/java/com/gluonhq/attach/ble/impl/AndroidBleService.java

```diff
@@ -60,22 +60,17 @@
     private static final ObservableList<BleDevice> devices = FXCollections.observableArrayList();
     private static final List<String> deviceNames = new LinkedList<>();
     private static final List<String> profileNames = new LinkedList<>();
-    private static boolean debug;
+    private static final boolean debug = Boolean.getBoolean(Constants.ATTACH_DEBUG);
 
     private static Consumer<ScanDetection> callback;
 
     static {
         LOG.fine("Loading AndroidBleService");
-        debug = Boolean.getBoolean(Constants.ATTACH_DEBUG);
         System.loadLibrary("Ble");
-        LOG.fine("Loaded AndroidBleService");
```

Comment:
why is this gone?

Reply:
There is a log right before loading the library. Seems unnecessary to have the second one.

Reply:
Second one is very important too. Loading library can go wrong.

Reply:
We had many issues before where loading a library failed, and if we don't have an indication that the library failed to load or succeeded, we're running blind.

Reply:
Restored